learn/what_is_meilisearch/telemetry.mdx

```diff
@@ -196,6 +196,8 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `pagination.max_total_hits`                        | The defined value for the `pagination.maxTotalHits` property                                | 1000
 | `faceting.max_values_per_facet`                    | The defined value for the `faceting.maxValuesPerFacet` property                             | 100
 | `distinct_attribute.set`                           | `true` if a field name is specified, otherwise `false`                                      | false
+| `proximity_precision.set`                          | `true` if the setting has been manually set, otherwise `false`.                             | `false`
+| `proximity_precision.value`                        | `byWord` or `byAttribute` if the setting has been set, otherwise `null`.                    | `null`
```

Comment:
I think this recently changed and the default is `byWord` instead of `null`. cc @ManyTheFish to confirm

Reply:
Yes you are right

Reply:
Thanks for bringing it up, @macraig—I wasn't sure whether the default change affected telemetry data.